tfhe/src/integer/server_key/comparator.rs

```diff
@@ -464,6 +629,69 @@ impl<'a> Comparator<'a> {
         RadixCiphertext { blocks }
     }
 
+    fn unchecked_scalar_min_or_max_parallelized<PBSOrder, T>(
+        &self,
+        lhs: &RadixCiphertext<PBSOrder>,
+        rhs: T,
+        selector: MinMaxSelector,
+    ) -> RadixCiphertext<PBSOrder>
+    where
+        PBSOrder: PBSOrderMarker,
+        T: DecomposableInto<u8>,
+    {
+        let (x_accumulator, y_accumulator) = match selector {
+            MinMaxSelector::Max => (&self.x_accumulator, &self.y_accumulator),
+            MinMaxSelector::Min => (&self.y_accumulator, &self.x_accumulator),
+        };
+
+        let mut mask = self.unchecked_scalar_sign_parallelized(lhs, rhs);
+        self.server_key
+            .key
+            .apply_lookup_table_assign(&mut mask, &self.mask_accumulator);
+
+        let message_modulus = self.server_key.key.message_modulus.0 as u64;
+        let bits_in_message = message_modulus.ilog2();
```

Comment:
eternal question of "should we manage weird moduli" ? I don't remember if we said something about it and would add support for non power of two moduli down the line in the end ? Sorry I know I already asked the question but I don't remember what we ended up deciding (if we did decide)

Reply:
yes indeed, I remember now ! thanks